.circleci/config.yml

```diff
@@ -24,6 +24,8 @@ jobs:
       - run: go get golang.org/x/tools/cmd/goimports
       - run: "! goimports ./... 2>&1 | read"
       - run: "! go mod tidy ./... 2>&1 | read"
+      - run: go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
can you pin to a version? client is on `1.16.0` https://github.com/keybase/client/blob/master/Jenkinsfile#L463, https://github.com/golangci/golangci-lint#ci-installation

Reply:
Done